example/swap2_double_integrator_3d_octomap.yaml

```diff
@@ -0,0 +1,18 @@
+environment:
+  min: [-5, -5, 0]
+  max: [5, 5, 5]
+  obstacles: 
+    - type: octomap
+      file: map
+      center: []
+      size: []
```

Comment:
should not be here in case of octomap

Reply:
Quim's code assumes they are there while reading the env file.